terraform/main_vm_start.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+TIME_LIMIT=1800
+
+# apt_install command_to_check package_name
+function apt_install {
+    while [[ ! `command -v $1` ]];
+    do
+        apt-get -y update
+        apt-get -y install $2
+        sleep 3
+    done
+}
+
+mkdir go
+export HOME=/
+cd $HOME/go
+echo "`date -u` - script start time" > $HOME/results.txt
+SECONDS=0
+
+apt_install pip3 python3-pip
+apt_install mysql mysql-client
+
+snap install --classic --channel=1.14/stable go
+
+export GOPATH=$HOME/go
+export PATH=/snap/bin:$GOPATH/bin:$PATH
+
+/snap/bin/go get -v -t github.com/Alexandr-TS/fleetspeak/...
+
+cd /go/src/github.com/Alexandr-TS/fleetspeak/
+git checkout prep_cloud
+
+ln -fs /usr/bin/python3 /usr/bin/python
+
+pip3 install -e fleetspeak_python/
+fleetspeak/build.sh
+
+gsutil cp fleetspeak/src/e2etesting/frr-master-server-main/frr_master_server_main ${storage_bucket_url}/bin/frr_master_server_main
+gsutil cp fleetspeak/src/client/client/client ${storage_bucket_url}/bin/client
+gsutil cp fleetspeak/src/server/server/server ${storage_bucket_url}/bin/server
+gsutil cp frr_python/frr_server.py ${storage_bucket_url}/frr_python/frr_server.py
```

Comment:
Probably it happens because clients and servers install fleetspeak and some other packages themselves using pip in their startup scripts. But I agree that it is not a good solution. It's fixed in the new commit.